.gitignore

```diff
@@ -18,3 +18,4 @@ built
 /lib/*/composer.lock
 /phpcs.xml
 /phpunit.xml
+*.sql
```

Comment:
Is this still relevant? If so maybe the path should be scoped to where the SQL files would be placed.

Reply:
I've found that I'm working with DB imports/exports when creating/editing a reference site, so adding this seems to make sense.

I'll scope it to the root folder.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/5162/commits/686e6c677837563924459a9c786fd96586c1768f